Subscriber/AdditionalAccessTokenClaimsAndHeaderSubscriber.php

```diff
@@ -29,14 +30,17 @@ public static function getSubscribedEvents(): array
 
     public function addClaims(JWTCreatedEvent $event): void
     {
+        $clock = new NativeClock(new \DateTimeZone('UTC'));
```

Comment:
It's more interesting to inject it into the construct like the example I shared and then make a call like these:
```php
$now = $this->clock->now();
```